src/js/program/program-controller.js

```diff
@@ -213,8 +213,7 @@ class ProgramController extends Eventable {
         this._setActiveMedia(castMediaController);
         // Initialize the provider last so it's setting properties on the (newly) active media model
         castMediaController.provider.init(playlistItem);
-        model.trigger('itemReady', playlistItem);
-
+        model.set('itemReady', true);
```

Comment:
Since we don't call `model.setActiveItem` there is no "playlistItem" event here. If we do need the "change: itemReady" event the you do have to set "itemReady" to false first (before `_setActiveMedia`)